emData/generate_MP.py

```diff
@@ -202,6 +202,8 @@ def FMRegion(region):
         topFile.write(
             "#pragma HLS resource variable=allstub->get_mem() latency=2\n"
             "#pragma HLS resource variable=instubdata.get_mem() latency=2\n"
+            "#pragma HLS resource variable=instubdata.get_mem_entries8A() latency=0\n"
+            "#pragma HLS resource variable=instubdata.get_mem_entries8B() latency=0\n"
```

Comment:
@jasonfan393 During C-synthesis, I see the following warnings:
```
WARNING: [TECH 200-23] The specified latency 0 on variable ('instubdata_nentries8A_V_load', /home/ahart/TrackFinding/jason/firmware-hls/TrackletAlgorithm/MatchProcessor.h:858->/home/ahart/TrackFinding/jason/firmware-hls/TopFunctions/ReducedCombinedConfig2/MatchProcessorTop.cc:29) is smaller than the min latency 1 of core 'RAM'. The min latency 1 is selected.
WARNING: [TECH 200-23] The specified latency 0 on variable ('instubdata_nentries8B_V_load', /home/ahart/TrackFinding/jason/firmware-hls/TrackletAlgorithm/MatchProcessor.h:859->/home/ahart/TrackFinding/jason/firmware-hls/TopFunctions/ReducedCombinedConfig2/MatchProcessorTop.cc:29) is smaller than the min latency 1 of core 'RAM'. The min latency 1 is selected.
```

Could you set the latency to one here, to avoid these warnings? I'm not sure if this affects the logic of the SectorProcessors that include the MP.